test/unit/config_test.rb

```diff
@@ -0,0 +1,417 @@
+
```

Comment:
I'm holding off on the `chefstyle/rubocop -a` PR until the final product, but am still willing to do the work premerge as it takes me very little time. I'll avoid any style issues in this PR for this reason.